### DISCUSSION TITLE
Lancement SYNAPPS

### DISCUSSION THREAD

**Discussion by Bgtb** [witsa/synapps#740] [Q&A] [answered]
Bonjour,
Je crée l'interface graphique d'une GTB existante sans synoptiques.
J'ai fait les mises à jour pour être en 2.5.8
J'ai publié mon synapps et importer tous mes synoptiques.
J'ai lié un utilisateur à mon synapps.
Toutefois, lorsque je me logue, synapps ne semble pas se lancer. J'ai juste 3 petit points qui s'affiche.

Sauriez-vous me dire s'il y a quelque part une coche pour valider le démarrage de synapps ??

Cordialement

**Comment by JeremieDENAEGEL** [comment, marked as answer]
Bonjour,
je ne sais pas si cela peut être la cause et du coup le résoudre.
Est-ce-que le serveur RIA de ton Redy est coché en "Hébergement RIA local" ?
(Menu Configuration/Système)
Non coché, c'est peut-etre la cause si tu es sur un réseau fermé

**Reply by Bgtb** [reply]
Super,
C'était ça le problème.
Merci beaucoup
